README.md

```diff
@@ -38,6 +38,12 @@ After creating a **YouTube Data API V3** access, you can create an API key which
 
 After that restart your server to enable the YouTube API access on your server.
 
+### Firebase Config
```

Comment:
This should also be optional, I can update the readme later.

The goal is to make this (mostly) work without Redis or Firebase or YouTube, if people don't want to set them up.